GTG/gtk/editor/editor.py

```diff
@@ -225,6 +242,78 @@ def init_accelerators(self):
         key, modifier = Gtk.accelerator_parse('<Control>q')
         agr.connect(key, modifier, Gtk.AccelFlags.VISIBLE, self.quit)
 
+    '''
+    TODO(jakubbrindza): Add the functionality to the existing calendar widgets.
+    This will require ammending and re-factoring the entire calendar.py.
+    '''
+
+    def show_popover_start(self, widget, event):
+        entry = self.builder.get_object("startdate_entry")
+        self.popover = self.builder.get_object("date_popover")
+        self.popover.set_relative_to(entry)
+        self.popover.set_modal(False)
+        self.popover.show_all()
```

Comment:
popover is not referenced from anywhere else than just this function. Make it local

```
popover = ...
```